### PR TITLE
Add support for text/html Accept header

### PR DESCRIPTION
**Pull request by alde**
If accessing a route with text/html in the Accept header render a HTML
page instead of returning a JSON.

### CODE REVIEW

File: handler/handler.go [hunk 1]
```diff
@@ -108,3 +113,65 @@ func writeJSON(status int, data interface{}, w http.ResponseWriter) error {
...
+	data["Memory"] = fmt.Sprintf("%.2f", task.TaskMem)
+
+	return data
+}
+
+// LabelColor is used to map a status to a label color
+func LabelColor(status string) string {
+	switch status {
+	case "TASK_FAILED":
+		return "red"
```
Comment (keis) [inline handler/handler.go]: colours feels more like a css thing. should the class perhaps just be "task-failed" or so?

Comment (alde) [inline handler/handler.go]: that is very true, maybe I should just export a ToLower function

**Comment by keis** [comment]
LGTM. Will give it a spin.

File: templates/error_404.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <div class="ui fixed inverted menu">
+            <div class="ui container">
+                <div class="header item">
+                    Eremetic
+                </div>
+            </div>
+        </div>
+        <div class="ui main container" style="padding-top: 50px;">
+            <div class="ui image content">
+                <img src="static/images/disappointed_minion.png">
```
Comment (keis) [inline templates/error_404.html]: guess this sholud be /static/..

Comment (alde) [inline templates/error_404.html]: yes! that's probably why it breaks on that one route

**Pull request merged**